integration_tests/conftest.py

```diff
@@ -0,0 +1,11 @@
+import json
+
+import pytest
+import requests
+
+
+@pytest.fixture(scope="module")
```

Comment:
This new fixture means every time a test module runs we're making a request. I see this is also defined below, so now both unit tests and integration tests rely on internet connectivity. Just pointing that out.
Do we need it per module, or could this fixture run per test run session only?

Reply:
@clavedeluna good catch. I did think about making it session scoped but I think we're running unit tests and integration tests in separate workflows so it would run twice anyway.